Makefile

```diff
@@ -261,7 +261,7 @@ cacheclean:
 	find theories plugins test-suite -name '.*.aux' -exec rm -f {} +
 
 cleanconfig:
-	rm -f config/Makefile config/coq_config.ml myocamlbuild_config.ml dev/ocamldebug-coq dev/camlp5.dbg config/Info-*.plist
+	rm -f config/Makefile config/coq_config.ml myocamlbuild_config.ml dev/ocamldebug-coq config/Info-*.plist
```

Comment:
Do we still use this myocamlbuild thing by the way?

Reply:
> Do we still use this myocamlbuild thing by the way?

#8754